File: src/main/Frontend/myApp/src/app/class/Shelter.ts
export class Shelter {
  get _id(): number {
    return this.id;
  }

  get _name(): string {
    return this.name;
  }

  get _phonenumber(): string {
    return this.phonenumber;
  }

  get _e_mail(): string {
    return this.e_mail;
  }

  get _city(): string {
    return this.city;
  }

  get _postcode(): string {
    return this.postcode;
  }

  get _adress(): string {
    return this.addres;
  }

  get _accoun_number(): string {
    return this.accoun_number;
  }





  public id: number = 0;
  public name: string = '';
  public phonenumber: string='';
  public e_mail: string = '';
  public city: string = '';
  public postcode: string = '';
  public addres: string = '';
  public accoun_number: string='';
  constructor() {
  }



}

File: src/main/java/com/example/demo/dal/entities/Shelter.java
package com.example.demo.dal.entities;


import lombok.*;

import javax.persistence.*;
@Entity
@Data
@AllArgsConstructor // minden argumentummla konstruktor
@NoArgsConstructor // ures konstruktor
@EqualsAndHashCode(callSuper = true) // legenerálja
@Table(name = "shelters") // tábla elnevezés
public class Shelter extends BaseEntity {

    private String name;

    private String city;

    private String addres;

    private String postcode;

    private String phonenumber;

    private String e_mail;

    private String accoun_number;

    private int stars;

    private int db;

    private String text;

}
